api/schemas.py

```diff
@@ -12,3 +12,17 @@ class PayloadSchema(Schema):
     date = fields.Date(required=True, validate=validate_date)
     hd = fields.Boolean(required=True)
     thumbs = fields.Boolean(missing=True)
+
+
+class CMEPayloadSchema(Schema):
+    startDate = fields.Date(format='%Y-%m-%d',
+                            required=True)
+    endDate = fields.Date(format='%Y-%m-%d',
+                          required=True)
+
+    @validates_schema()
+    def validate_cme_period(self, data, **kwargs):
+        if not (data['startDate'] <= data['endDate']):
```

Comment:
Can we rewrite this condition without using negation to improve code readability?

Reply:
https://github.com/mstoro/nasa_api/pull/3/commits/1ee43f97545851be885052bf454acdd6ed5dc6c9